spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/pom.xml

```diff
@@ -2,16 +2,31 @@
 <project xmlns="https://maven.apache.org/POM/4.0.0"
          xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
```

Comment:
If we use `spring-boot-starter-parent`, don't we lose the ability to enforce checkstyle rules, etc?

Reply:
Yes unfortunately checkstyle is lost. Seems like this is the price for disconnecting from the parent.